### PR TITLE
Material UI

### PR DESCRIPTION
**Pull request by Ishan09811**

### CODE REVIEW

File: src/pandroid/app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -7,6 +7,14 @@
     android:layout_height="match_parent"
     tools:context=".app.MainActivity">
 
+    <com.google.android.material.appbar.MaterialToolbar
```
Comment (GabrielBRDeveloper) [inline src/pandroid/app/src/main/res/layout/activity_main.xml:10]: Remove header bar

File: src/pandroid/app/src/main/res/layout/dialog_lua_scripts.xml [hunk 2]
```diff
@@ -31,35 +30,35 @@
 
         </androidx.constraintlayout.widget.ConstraintLayout>
 
-        <androidx.appcompat.widget.LinearLayoutCompat
+        <LinearLayout
```
Comment (GabrielBRDeveloper) [inline src/pandroid/app/src/main/res/layout/dialog_lua_scripts.xml:33]: Use LinearLayoutCompat:
https://developer.android.com/jetpack/androidx?hl=pt-br

File: src/pandroid/app/src/main/res/layout/fragment_game_drawer.xml [hunk 3]
```diff
@@ -30,9 +31,9 @@
                 android:layout_height="match_parent"
                 android:background="?colorSurface"/>
 
-        </androidx.cardview.widget.CardView>
+        </com.google.android.material.card.MaterialCardView>
 
-        <androidx.appcompat.widget.LinearLayoutCompat
+        <LinearLayout
```
Comment (GabrielBRDeveloper) [inline src/pandroid/app/src/main/res/layout/fragment_game_drawer.xml:36]: Use LinearLayoutCompat:
https://developer.android.com/jetpack/androidx?hl=pt-br

File: src/pandroid/app/src/main/res/layout/fragment_game_drawer.xml [hunk 4]
```diff
@@ -58,16 +59,16 @@
...
-    </androidx.appcompat.widget.LinearLayoutCompat>
+    </LinearLayout>
 
     <ScrollView
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:layout_marginTop="10dp">
 
-        <androidx.appcompat.widget.LinearLayoutCompat
+        <LinearLayout
```
Comment (GabrielBRDeveloper) [inline src/pandroid/app/src/main/res/layout/fragment_game_drawer.xml:71]: Use LinearLayoutCompat:
https://developer.android.com/jetpack/androidx?hl=pt-br

Comment (Ishan09811) [reply src/pandroid/app/src/main/res/layout/fragment_game_drawer.xml:71]: LinearLayoutCompat is mostly used for android 4.0 or below so it's doesn't matter in this android 7.0 or above

**Pull request merged**